test/projects/dune

```diff
@@ -25,6 +25,7 @@
 
 (rule
  (alias runtest)
+ (enabled_if (<> %{os_type} Win32))
  (action
   (diff outside_detected_project.expected outside_detected_project.output)))
```

Comment:
Missing `enabled_if` on the rule for that file. Also, why is this one not working on Windows?

Reply:
I added the clause on the related rule